modules/deploying-models-on-single-node-openshift-using-kserve-raw-deployment-mode.adoc

```diff
@@ -1,15 +1,11 @@
 :_module-type: PROCEDURE
 
 [id="deploying-models-on-single-node-openshift-using-kserve-raw-deployment-mode_{context}"]
-= Deploying models on single node {openshift-platform} using KServe raw deployment mode
+= Preparing your cluster for standard mode deployment
 
 [role='_abstract']
-You can deploy a machine learning model by using KServe raw deployment mode on single node {openshift-platform}. Raw deployment mode offers several advantages over Knative, such as the ability to mount multiple volumes.
 
-[IMPORTANT]
-====
-Deploying a machine learning model using KServe raw deployment mode on single node OpenShift is a Limited Availability feature. Limited Availability means that you can install and receive support for the feature only with specific approval from the {org-name} AI Business Unit. Without such approval, the feature is unsupported.
-====
+You can deploy a machine learning model by using standard deployment mode, which is based on KServe RawDeployment mode on single node {openshift-platform}. Standard mode offers several advantages over advanced, such as the ability to mount multiple volumes. 
```

Comment:
RawDeployment for standard OpenShift is going GA. But I don't think single-node has been fully tested nor it is ready for GA. So:

```suggestion
You can deploy a machine learning model by using standard deployment mode, which is based on KServe RawDeployment mode on {openshift-platform}. Standard mode offers several advantages over advanced, such as the ability to mount multiple volumes. 
```